package-lock.json

```diff
@@ -1,8 +1,1056 @@
 {
-  "name": "ejs-challenge",
+  "name": "Blogify",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "Blogify",
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "body-parser": "^1.20.0",
+        "dotenv": "^16.0.3",
+        "ejs": "^2.7.4",
+        "express": "^4.18.1",
+        "lodash": "^4.17.21",
+        "mongoose": "^6.5.3"
+      }
+    },
+    "node_modules/@types/node": {
```

Comment:
Could you please remove the unnecessary code from this file!

Reply:
@tusharkhatriofficial I  guess it is not unnecessary code. It is automatically generated on installing dotenv package

Reply:
Ya I just checked it.